Source/Scene/GroundPolylinePrimitive.js

```diff
@@ -0,0 +1,856 @@
+
+            var vsPickMorphSource = ShaderSource.createPickVertexShaderSource(vsMorph);
+            vsPickMorphSource = Primitive._updatePickColorAttribute(vsPick);
+
+            var vsPick3D = new ShaderSource({
+                defines : vsDefines,
+                sources : [vsPick]
+            });
+            var fsPick3D = new ShaderSource({
+                defines : fsDefines.concat(['PICK']),
+                sources : [PolylineShadowVolumeFS],
+                pickColorQualifier : 'varying'
+            });
+
+            groundPolylinePrimitive._spPick = ShaderProgram.replaceCache({
+                context : context,
+                shaderProgram : groundPolylinePrimitive._spPick,
+                vertexShaderSource : vsPick3D,
+                fragmentShaderSource : fsPick3D,
+                attributeLocations : attributeLocations
+            });
+
+            // Derive 2D/CV
+            var pickProgram2D = context.shaderCache.getDerivedShaderProgram(groundPolylinePrimitive._spPick, '2dPick');
+            if (!defined(pickProgram2D)) {
+                var vsPick2D = new ShaderSource({
+                    defines : vsDefines.concat(['COLUMBUS_VIEW_2D']),
+                    sources : [vsPick]
+                });
+                pickProgram2D = context.shaderCache.createDerivedShaderProgram(groundPolylinePrimitive._spPick, '2dPick', {
+                    context : context,
+                    shaderProgram : groundPolylinePrimitive._spPick2D,
+                    vertexShaderSource : vsPick2D,
+                    fragmentShaderSource : fsPick3D,
+                    attributeLocations : attributeLocations
+                });
+            }
+            groundPolylinePrimitive._spPick2D = pickProgram2D;
+
+            // Derive Morph
+            var pickProgramMorph = context.shaderCache.getDerivedShaderProgram(groundPolylinePrimitive._spPick, 'MorphPick');
+            if (!defined(pickProgramMorph)) {
+                var vsPickMorph = new ShaderSource({
+                    defines : vsDefines,
+                    sources : [vsPickMorphSource]
+                });
+                var fsPickMorph = new ShaderSource({
+                    defines : ['PICK'],
+                    sources : [PolylineShadowVolumeMorphFS],
+                    pickColorQualifier : 'varying'
+                });
+                pickProgramMorph = context.shaderCache.createDerivedShaderProgram(groundPolylinePrimitive._spPick, 'MorphPick', {
+                    context : context,
+                    shaderProgram : groundPolylinePrimitive._spPickMorph,
+                    vertexShaderSource : vsPickMorph,
+                    fragmentShaderSource : fsPickMorph,
+                    attributeLocations : attributeLocations
+                });
+            }
+            groundPolylinePrimitive._spPickMorph = pickProgramMorph;
+        } else {
+            groundPolylinePrimitive._spPick = ShaderProgram.fromCache({
```

Comment:
Remove this. If `allowPicking` is false, don't submit any commands. The only reason to submit a command without it being picked would be to lay down depth.